src/components/form/index.js

```diff
@@ -24,7 +24,7 @@ const useStyles = makeStyles((theme) => ({
 }))
 
 const patterns = {
-    discord: /^https:\/\/discord\.com\/api\/webhooks\//,
+    discord: /^https:\/\/discord(app)*\.com\/api\/webhooks\//,
```

Comment:
Interesting. Don't know why discord would have two patterns for webhook URLs but good find.